### DISCUSSION TITLE
Facing issue of extra worker getting created when upgraded WebdriverIo from V7 to V8

### DISCUSSION THREAD

**Discussion by shivani97ksp** [webdriverio/webdriverio#10915] [Community]
Hello Team,

Recently I upgraded wdio from version 7.26 to version 8.14, I am able to execute feature files, but two workers are getting created everytime even though providing one feature file in specs with maxinstance of 1. And after the execution of all scenarios once from the feature file, it starts to run again the same feature file once more.
It used to work completely fine with the older version, please provide me any insights for this issue. 
Dependencies used in package.json:
"dependencies": {
    "@wdio/cli": "^8.14.4"
  },
  "devDependencies": {
    "@wdio/allure-reporter": "^8.14.0",
    "@wdio/cucumber-framework": "^8.14.0",
    "@wdio/local-runner": "^8.14.3",
    "@wdio/spec-reporter": "^8.14.0",
    "allure-commandline": "2.23.1",
    "chai": "^4.3.7",
    "chromedriver": "^115.0.1",
    "jsdom": "22.1.0",
    "jsdom-global": "3.0.2",
    "ts-node": "^10.9.1",
    "typescript": "^5.1.6",
    "wdio-chromedriver-service": "^8.1.1"
  }

**Comment by christian-bromann** [comment]
@shivani97ksp can you provide a minimal reproducible example?

**Comment by shivani97ksp** [comment]
@christian-bromann It's working fine now, there was casing issue in specs feature file path. That never created issue in version 7 but in version 8, seems to have case sensitive. Thanks.

There is one more issue I am currently facing, after going through the [blog](https://webdriver.io/blog/2023/07/31/driver-management) when I commented services in config file, getting error ERROR @wdio/runner: Error: unable to get local issuer certificate
[0-0]     at TLSSocket.onConnectSecure (node:_tls_wrap:1540:34)
[0-0]     at TLSSocket.emit (node:events:513:28)
[0-0]     at TLSSocket.emit (node:domain:489:12)
[0-0]     at TLSSocket._finishInit (node:_tls_wrap:959:8)
[0-0]     at TLSWrap.ssl.onhandshakedone (node:_tls_wrap:743:12)

The same issue I faced even when I did setup for a new project using command npm init wdio . and then I ran npm run wdio, faced same issue.
This new project has package.json with dependencies:
"devDependencies": {
    "@wdio/allure-reporter": "^8.14.6",
    "@wdio/cli": "^8.14.6",
    "@wdio/cucumber-framework": "^8.14.6",
    "@wdio/local-runner": "^8.14.6",
    "@wdio/spec-reporter": "^8.14.0",
    "ts-node": "^10.9.1",
    "typescript": "^5.1.6"
  },
And in config:
capabilities: [{
        browserName: 'chrome'
    }],
//services: [],
Can you tell me why I am facing this issue even with new setup?
